common/djangoapps/third_party_auth/tests/utils.py

```diff
@@ -139,3 +141,9 @@ def prepare_saml_response_from_xml(xml, relay_state='testshib'):
         relay_state=OneLogin_Saml2_Utils.escape_url(relay_state),
         saml_response=OneLogin_Saml2_Utils.escape_url(b64encoded_xml)
     )
+
+
+def skip_tpa_tests():
+    if AUTH_FEATURE_ENABLED:
```

Comment:
docstring needed?

Reply:
hmm oh yeah surprised I didn't see that error. Probably is what's failing now.